src/AppKit/NSFont.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Runtime.InteropServices;
+using XamCore.CoreFoundation;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.CoreGraphics;
+using XamCore.CoreAnimation;
+using XamCore.CoreText;
+
+using CGGlyph = System.UInt16;
+
+namespace XamCore.AppKit {
+	public partial class NSFont
+	{
+		public static NSFont FromCTFont (CTFont font)
```

Comment:
@spouliot and/or @rolfbjarne - CTFont and NSFont are toll free bridged and we did not have a way I could see to leverage this. Is this the correct/safe way of handling this?